lib/msal-common/test/client/AuthorizationCodeClient.spec.ts

```diff
@@ -335,7 +335,8 @@ describe("AuthorizationCodeClient unit tests", () => {
                 scopes: [...TEST_CONFIG.DEFAULT_GRAPH_SCOPE, ...TEST_CONFIG.DEFAULT_SCOPES],
                 redirectUri: TEST_URIS.TEST_REDIRECT_URI_LOCALHOST,
                 code: TEST_TOKENS.AUTHORIZATION_CODE,
-                codeVerifier: TEST_CONFIG.TEST_VERIFIER
+                codeVerifier: TEST_CONFIG.TEST_VERIFIER,
+                claims: "{claimsRequest}"
```

Comment:
Is this what a typical claims request looks like? I'd be interested for a real-world example (out of curiosity).

Reply:
Usually it is serialized JSON object. Here is how claims request looks like when supporting Long Lived Tokens:
"{"access_token":{"xms_cc":{"values":["CP1"]}}}"